<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard::CCassetteNumberInputWidget</class>
 <widget class="QWidget" name="Dashboard::CCassetteNumberInputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="MainMenu::CWheelPanel" name="scrollPanelCassetteNumber" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>331</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>102</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainMenu::CWheelPanel</class>
   <extends>QWidget</extends>
   <header>MainMenu/Include/WheelPanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
